src/test/java/org/apache/commons/lang3/JavaVersionTest.java

```diff
@@ -72,4 +73,28 @@ public void testToString() {
         assertEquals("1.2", JAVA_1_2.toString());
     }
 
+    @Test
+    public void testCurrent() throws Exception {
+        if (SystemUtils.JAVA_SPECIFICATION_VERSION.equals("1.1")) {
+            assertEquals(JAVA_1_1, JavaVersion.CURRENT);
+        } else if (SystemUtils.JAVA_SPECIFICATION_VERSION.equals("1.2")) {
+            assertEquals(JAVA_1_2, JavaVersion.CURRENT);
+        } else if (SystemUtils.JAVA_SPECIFICATION_VERSION.equals("1.3")) {
+            assertEquals(JAVA_1_3, JavaVersion.CURRENT);
+        } else if (SystemUtils.JAVA_SPECIFICATION_VERSION.equals("1.4")) {
+            assertEquals(JAVA_1_4, JavaVersion.CURRENT);
+        } else if (SystemUtils.JAVA_SPECIFICATION_VERSION.equals("1.5")) {
+            assertEquals(JAVA_1_5, JavaVersion.CURRENT);
+        } else if (SystemUtils.JAVA_SPECIFICATION_VERSION.equals("1.6")) {
+            assertEquals(JAVA_1_6, JavaVersion.CURRENT);
+        } else if (SystemUtils.JAVA_SPECIFICATION_VERSION.equals("1.7")) {
+            assertEquals(JAVA_1_7, JavaVersion.CURRENT);
+        } else if (SystemUtils.JAVA_SPECIFICATION_VERSION.equals("1.8")) {
+            assertEquals(JAVA_1_8, JavaVersion.CURRENT);
+        } else if (SystemUtils.JAVA_SPECIFICATION_VERSION.equals("9.0.0")) {
```

Comment:
Not sure if this is correct. Shouldn't it be something like `9.` if I understand 

http://openjdk.java.net/jeps/223

correctly?